build-env.sh

```diff
@@ -0,0 +1,16 @@
+SERVER_HOST=""
+SERVER_PORT=""
+DEFAULT_SOCKET_PATH=""
```

Comment:
This "path" represents a namespace, instead of a path.
This will not be configurable, its more like a business related param.

I will remove this from the `.env` file and go back to "/" as the default namespace, but I think we should figure out a better configuration for our namespaces, every event is currently running on a "root" namespace, lets figure out a way to improve this behavior.

Reply:
The name `PATH` will be replaced with `NAMESPACE`.
Ref: https://github.com/googollee/go-socket.io/blob/88e44eae2d1d36c9e852a7e7ced2b30e89ffd64c/server.go#L38